chef/cookbooks/crowbar/recipes/default.rb

```diff
@@ -231,7 +231,15 @@
   action :create
 end
 
-unless node["crowbar"].nil? or node["crowbar"]["users"].nil? or node["crowbar"]["realm"].nil?
+if !node["crowbar"].nil? && !node["crowbar"]["realm"].nil?
+  # After installation of a gem, we have a new path for the new gem, so we
+  # need to reset the paths if we can't load the gem
+  begin
+    require "inifile"
+  rescue LoadError
+    Gem.clear_paths
```

Comment:
Pardon my ruby ignorance, but does this mean that after `Gem.clear_paths`, `inifile` can be succesfuly used later in this block?

Reply:
Yes